runtime/src/runtime.rs

```diff
@@ -0,0 +1,443 @@
+// Copyright 2019-2020 PureStake Inc.
+// This file is part of Moonbeam.
+
+// Moonbeam is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Moonbeam is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Moonbeam.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Exposes the core runtime types.
+//! Feature `standalone` allows to configure the runtime for a standalone node instead of a parachain.
+
+use crate::{
+	opaque, AccountId, Balance, Block, Box, EthereumFindAuthor, Index, UncheckedExtrinsic, Vec,
+};
+use fp_rpc::TransactionStatus;
+use frame_support::{construct_runtime, traits::FindAuthor, ConsensusEngineId};
+
+use frame_support::traits::{Get, Randomness};
+use pallet_evm::{Account as EVMAccount, FeeCalculator, Runner};
+use sp_api::impl_runtime_apis;
+use sp_core::{OpaqueMetadata, H160, H256, U256};
+use sp_runtime::{
+	create_runtime_str,
+	traits::Block as BlockT,
+	transaction_validity::{TransactionSource, TransactionValidity},
+	ApplyExtrinsicResult,
+};
+use sp_std::prelude::*;
+use sp_version::RuntimeVersion;
+
+/// Executive: handles dispatch to the various modules.
+pub type Executive = frame_executive::Executive<
+	Runtime,
+	Block,
+	frame_system::ChainContext<Runtime>,
+	Runtime,
+	AllModules,
+>;
+
+#[cfg(not(feature = "standalone"))]
+pub use parachain::*;
+#[cfg(feature = "standalone")]
+pub use standalone::*;
+
+#[cfg(not(feature = "standalone"))]
+pub const NAME: &str = "moonbase-alphanet";
+#[cfg(feature = "standalone")]
+pub const NAME: &str = "moonbase-standalone";
+
+/// This runtime version.
+pub const VERSION: RuntimeVersion = RuntimeVersion {
+	spec_name: create_runtime_str!(NAME),
+	impl_name: create_runtime_str!(NAME),
+	authoring_version: 3,
+	spec_version: 9,
+	impl_version: 1,
+	apis: RUNTIME_API_VERSIONS,
+	transaction_version: 2,
+};
+
+#[cfg(not(feature = "standalone"))]
```

Comment:
This was our original designed and decided to move to 2 separate files for parachain/standalone, mostly because it made it easier to know what was in the parachain vs what was in the standalone. Having it in a single file with multiple cfg macros make it header to read in my opinion.

Reply:
Let's get others opinion on that

Reply:
Here's some context https://github.com/PureStake/moonbeam/pull/60#issuecomment-707617988

I don't have a strong opinion either way. But I do hope that eventually (maybe in the next 6-12 months??) we can just have a single runtime (no features) that runs on either the parachain or standalone chasis.

Reply:
I wanted to put the cf attribute directly on the member declaration (Aura/Grandpa/ParachainInfo/ParachainUpgrade) but it don't work.

Otherwise I can make a small macro to declare in one place what are the common, standalone-only and parachain-only pallets, but it might be overkill. The runtime declaration is quite short, and I sorted all entries and separated the common pallets from the dedicated ones. 

Otherwise we can make submodules for the standalone and parachain and reintroduce the macros, while keeping their invocation in the `runtime` mod.

The main goal here is to have the runtime in the `runtime` mod to have better namespacing of the types.